packages/cli/src/commands/accounts/add.ts

```diff
@@ -0,0 +1,46 @@
+import {Command} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import * as Heroku from '@heroku-cli/schema'
+import {add, list} from '../../lib/accounts/accounts'
+
+export default class Add extends Command {
```

Comment:
In the plugin, there's an `sso` flag that is required to facilitate login for an sso user, and I don't see that here. Actually, I don't see us performing the login in the new command.. did we change the behavior here? Like, with the plugin, if a user is not logged in, we're starting the login for them I think. Here, I think we are just telling them they need to log in. Is that right?

Reply:
Yes, I made that behavior change here. The actual login command has several flags that I did not want to reintroduce here, beyond just sso. We could do this a different way and start the login process here, but I thought it would be better to keep this command focused on adding an account.

Reply:
Actually, I could get rid of the {retry: false} in the API call and it would prompt the user to login. I'm not sure why I added that, but it's not necessary. I'm going to remove it and then I can remove some of the error logic as well. If you're not logged in, it should prompt you to log in the way most of the rest of our commands do.